examples/kubernetes/README.rst

```diff
@@ -0,0 +1,58 @@
+Kubernetes Deployment
+=====================
+
+This directory contains all Cilium deployment files that can be used in
+Kubernetes.
+
+Each directory represents a Kubernetes version, from :code:`1.7` to :code:`1.11`,
+and inside each version there is a list of files to deploy Cilium.
+
+The structure directory will be :code:`${k8s_major_version}.${k8s_minor_version}/*.yaml`.
+
+To generate those files simply run :code:`make`, which will pick up the Cilium
+version from the :code:`VERSION` file at the root of the repo, or if you want to
```

Comment:
Nit: repo --> repository . Would be good to use more formal full words instead of abbreviations in the documentation.

Reply:
Done